index.html

```diff
@@ -7,29 +7,46 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title> Portfolio </title>
   <link rel="stylesheet" href="./style.css">
+  
 </head>
-
 <body>
   <main>
     <header>
       <nav>
         <div class="desktop-menu">
           <ul>
             <li>
-              <a href="">Portfolio</a>
+              <a href="#">Portfolio</a>
             </li>
             <li>
-              <a href="">About</a>
+              <a href="#">About</a>
             </li>
             <li>
-              <a href="">Contact</a>
+              <a href="#">Contact</a>
             </li>
           </ul>
         </div>
-        <div class="mobile-menu-icon">
-          <img src="./images/Union.svg" alt="Bar Image">
+      </nav>
+      <nav class="navbar">
+        
+        <ul class="nav-menu">
+          <li class="nav-item">
+            <a href="#" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#" class="nav-link">Contact</a>
```

Comment:
- [ ] Good work making the design changes. However, kindly ensure that when clicking on a link, a specified section of the page is displayed. This could be achieved by adding a pathname that Is the id of the corresponding sections of the page to the `<a>` tags.